build-hazelcast-homebrew-package.sh

```diff
@@ -63,25 +63,24 @@ sed -i "s+conflicts_with \".*\"$+conflicts_with \"$CONFLICTS\"+g" "${HZ_DISTRIBU
 if [[ ! ( ${HZ_VERSION} =~ ^.*+(SNAPSHOT|BETA|DR).*^ ) ]]; then
   HZ_MINOR_VERSION=$(echo "${HZ_VERSION}" | cut -c -3)
 
-  rm -f "Aliases/${HZ_DISTRIBUTION}-${HZ_MINOR_VERSION}"
-  ln -s "../${HZ_DISTRIBUTION}@${BREW_PACKAGE_VERSION}.rb" "Aliases/${HZ_DISTRIBUTION}-${HZ_MINOR_VERSION}"
+  rm -f "${HZ_DISTRIBUTION}-${HZ_MINOR_VERSION}.rb"
```

Comment:
Do we actually need to remove this file if we overwrite it in the next line? Same on line 82